jcl/source/windows/JclDebug.pas

```diff
@@ -4146,6 +4285,62 @@ function TJclDebugInfoExports.GetLocationInfo(const Addr: Pointer; out Info: TJc
   end;
 end;
 
+function TJclDebugInfoExports.GetAddress(const UnitName : string; const ProcName : String): Pointer;
+var
+  I, BasePos: Integer;
+  Desc      : TJclBorUmDescription;
+  RawName   : Boolean;
+  tUnitName : string;
+  Unmangled : string;
+  B         : Boolean;
+begin
+  Result := nil;
+  {$IFDEF BORLAND}
+  RawName := not FImage.IsPackage;
+  {$ENDIF BORLAND}
+  {$IFDEF FPC}
+  RawName := True;
+  {$ENDIF FPC}
+  with FImage.ExportList do
+  begin
+//    SortList(esAddress, False);
```

Comment:
Applies to entire method, same as above:
no type indenting, no prefix on variables, begin/end indentation...